### PR TITLE
Adds block about PHP-FPM behaviour

### PR DESCRIPTION
**Pull request by Eydamos**
Related to the issue https://github.com/symfony/cli/issues/178 I added some documentation on the behaviour of the PHP-FPM in the webserver.
Even though the behaviour is currently "expected" it is not obvious for the user.

### CODE REVIEW

**Review by dbrumann — APPROVED** [review]
I like it, but I am not sure everyone will get what the tip is trying to warn about from the current wording. Maybe we should add something like, "This means when an index.html file is present in \`\`public\`\`, it will be displayed instead of your Symfony application."

**Comment by TimWerdin** [comment]
@dbrumann I have improved the message of the tip to make the behaviour clearer
Please recheck

**Review by dbrumann — APPROVED** [review]

File: setup/symfony_server.rst [hunk 1]
```diff
@@ -57,6 +57,22 @@ run the Symfony server in the background:
     # show the latest log messages
     $ symfony server:log
 
+Enabling PHP-FPM
+----------------
+
+When the server starts it will check the ``public`` folder for an ``index.php``
+file. If this file is found the server will automatically start with PHP-FPM
```
Comment (wouterj) [inline setup/symfony_server.rst:64]: @fabpot can you please check the first sentence? From your comment in the referenced issue, I get the feeling that more front-controller files are checked than just `public/index.php`?

Comment (fabpot) [reply setup/symfony_server.rst:64]: Yes, it checks for "common" patterns like `web/app.php`, `web/app_dev.php` or `public/index.php`.

File: setup/symfony_server.rst [hunk 2]
```diff
@@ -57,6 +57,22 @@ run the Symfony server in the background:
...
+enabled. Otherwise the server will start without PHP-FPM and will show a
+``Page not found`` page when trying to access a ``.php`` file in the browser.
+
+.. tip::
+
+    When an index.html and an index.php file are present the server will still
```
Comment (fabpot) [inline setup/symfony_server.rst:70]: Missing quotes around index.html and index.php

**Review by fabpot — CHANGES_REQUESTED** [review]

File: setup/symfony_server.rst (hunk 2 above)
Comment (TimWerdin) [reply setup/symfony_server.rst]: Added quotes

File: setup/symfony_server.rst (hunk 1 above)
Comment (TimWerdin) [reply setup/symfony_server.rst]: I have changed the text to mention those common patterns

**Review by fabpot — APPROVED** [review]

**Pull request merged**